modules/ROOT/partials/events/tinymcespellchecker-events.adoc

```diff
@@ -9,4 +9,5 @@ The following events are provided by the xref:introduction-to-tiny-spellchecker.
 |SpellcheckStart |N/A |Fired when spellchecking is enabled.
 |SpellcheckEnd |N/A |Fired when spellchecking is disabled.
 |SpellcheckerLanguageChanged |`+{ language: string, prevLanguage: string }+` |Fired when the spellchecking language is changed.
+|SpellcheckerUpdated |`+{ spelling: SpellingUpdate }+` |Fired when the editor content is checked for misspellings and suggestions.
```

Comment:
The conjunction, _and_, means _suggestions_ is, so far as I can work out, incorrect here.

Re-writing so it is explicit (rather than implicit) which two phrases are being joined makes it more obvious:

_Fired when the editor content is checked for misspellings. Also fired when the editor content is checked for suggestions._

As I understand it, Spell Checker Pro doesn’t *check* for suggestions, it *makes* suggestions.

Making a semi-educated guess, _and suggestions_ is, here, meant to infer the creation and adding of the spelling objects to the event log.

A suggested re-write, assuming my semi-educated guess is corect:

```suggestion
|SpellcheckerUpdated |`+{ spelling: SpellingUpdate }+` |Fired when editor content is checked for misspellings. The event log entry also includes xref:introduction-to-tiny-spellchecker.adoc#the-spelling-object[`+spelling+`] objects containing the spell check results.
```

The edit above assumes a supporting edit in `introduction-to-tiny-spellchecker.adoc`. To wit:

```diff
--- introduction-to-tiny-spellchecker.adoc	2023-05-30 08:14:02
+++ introduction-to-tiny-spellchecker-bforte-edit.adoc	2023-05-30 08:14:35
@@ -240,6 +240,7 @@
 });
 ----
 
+[the-spelling-object]
 ==== The `spelling` object
 
 The `+spelling+` object, provided by the `+SpellcheckerUpdated+` event, contains the result of the spellcheck by the spelling service. The keys of the outermost object represent the language code for each language. Under each language code, there is an object with keys representing misspelled words and values that are arrays of possible correct spellings.
```

Reply:
The checks are separate requests, though both occur during the same "check" of the content which also fires the event. The content is first checked for misspellings, and then suggestions are retrieved for those words, and once those suggestions are retrieved the event is fired, and includes those suggestions.

I am wrong though, the editor content is not what is checked for suggestions :joy:, I'll update to say
```suggestion
|SpellcheckerUpdated |`+{ spelling: SpellingUpdate }+` |Fired when the editor content is checked for misspellings, and includes suggestions within xref:introduction-to-tiny-spellchecker.adoc#spelling-object[the `+spelling+` object].
```